wgpu-core/src/command/render.rs

```diff
@@ -508,6 +508,25 @@ struct RenderPassInfo<'a, B: hal::Backend> {
     _phantom: PhantomData<B>,
 }
 
+#[repr(packed)]
+#[allow(dead_code)]
+struct DrawIndirectBuffer {
+    vertex_count: u32,
+    instance_count: u32,
+    first_vertex: u32,
+    first_instance: u32,
+}
+
+#[repr(packed)]
+#[allow(dead_code)]
+struct DrawIndexedIndirectBuffer {
+    index_count: u32,
+    instance_count: u32,
+    first_index: u32,
+    base_vertex: u32,
```

Comment:
One of these members (I think it is this one) is a i32 not a u32. Check some spec (likely DX12 docs)

Reply:
Well shouldn't matter what DX12 does but you're right, Webbgpu specs baseVertex as signed!

Reply:
Only mentioned DX as I knew they had good docs on it, all indirect args are the same cross api.